src/config.ts

```diff
@@ -10,68 +10,26 @@
  * All portions of the code written by Atypon Systems LLC are Copyright (c) 2019 Atypon Systems LLC. All Rights Reserved.
  */
 
-const isTrue = (value: string | undefined) => {
-  return value === '1' || value === 'true'
-}
-
-const normalizeURL = (url: string | undefined) => {
-  return url && url.replace(/\/$/, '')
-}
-
-interface Config {
+export interface Config {
```

Comment:
Let's call this `ManuscriptsEditorConfig`. We should be able to use this in the parent app so to avoid duplicating the type definition.

Reply:
Done